libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/README.md

```diff
@@ -0,0 +1,17 @@
+# BOLD Carbon CH₄ – Baseline Emission Factors
```

Comment:
I think this processor is much more than _Baseline Emission Factors_, I would go with name _Avoided Emission Calculation_ or something similar.